### PR TITLE
[cypress] update overview feature for pf update

### PR DESCRIPTION
**Pull request by jshaughn**
Epic #4260

### CODE REVIEW

File: frontend/cypress/integration/featureFiles/overview.feature [hunk 1]
```diff
@@ -33,15 +33,15 @@ Feature: Kiali Overview page
...
+    And user doesn't see the "beta" namespace card
 
   @overview-page
   Scenario: Filter by health
     When user filters "Failure" health
     Then user sees the "alpha" namespace
-    And user sees the "beta" namespace
-    And user doesn't see the "default" namespace
+    And user sees the "beta" namespace card
+    And user doesn't see the "default" namespace ca
```
Comment (lucasponce) [inline frontend/cypress/integration/featureFiles/overview.feature:44]: ca -> card ?

**Review by lucasponce — CHANGES_REQUESTED** [review]
A couple of changes
https://github.com/jshaughn/kiali/pull/12

Note that I see a constant crash in cypress GUI on this test.
But the headless test worked fine.

File: frontend/cypress/integration/featureFiles/overview.feature (hunk 1 above)
Comment (jshaughn) [reply frontend/cypress/integration/featureFiles/overview.feature:44]: Weird, not sure how I managed to clip chars.

**Comment by jshaughn** [comment]
> Note that I see a constant crash in cypress GUI on this test.  But the headless test worked fine.

This is true for me as well.  I don't know whey, but browser always crashes with an, "Oh snap!"    If I run a few tests at a time it is fine, it is something cumulative.

Merged in your PR.

**Pull request merged**